deploy/helm/kubecf/assets/operations/instance_groups/eirini-helm.yaml

```
@@ -0,0 +1,116 @@
+{{- if and (.Values.features.eirini.enabled) (.Values.features.eirini.use_helm_release) }}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/opi?/enabled?
+  value: true
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/opi?/url?
+  value: "https://eirini-opi.{{ .Release.Namespace }}.svc.cluster.local:8085"
```

The cluster domain, in this case `cluster.local`, can be set to something else in different clusters.
To increase compatibility, you have to leave `cluster.local` out.

```suggestion
  value: "https://eirini-opi.{{ .Release.Namespace }}.svc:8085"
```

Just let the DNS server resolve it for you.